### ISSUE TITLE
Prefix Composer dependencies

### ISSUE THREAD

**Issue by swissspidy** [10up/plugin-check#328]
PHPCS is installed via Composer and we use the Composer autoloader for loading all these classes.

To avoid conflicts with any other plugins bundling PHPCS, we should prefix the dependencies' classes with something like PHP-Scoper. See https://github.com/googleforcreators/web-stories-wp/ and https://github.com/google/site-kit-wp for some prior art where this is done.

**Comment by ernilambar** [comment]
```
vendor/afragen/wordpress-plugin-readme-parser/class-parser.php
```

> That's being loaded by plugin-check even on WordPress.org, that shouldn't be happening. The WordPress.org file should take precedence. I guess the plugin-check auto-loader is simply being hit first.

Meta team has been using hacky approach for the deployment of the plugin due to this issue and wants PCP plugin to handle the loading of parser class in itself.

Suggestions are welcome so that we can fix the loading issue.

**Comment by ernilambar** [comment]
Alternative approach would be having readme parser as internal package. We just change the PHP namespace. Since that class changes very less frequently, it would not be much of burden to sync that one file to our repo. We could either have automated syncing or develop internal tool inside the repo.

**Comment by swissspidy** [comment]
We still have PHPCS as a dependency too, but probably not really common in other plugins.

**Comment by ernilambar** [comment]
Yes, agree. We should somehow fix the related PR for PHP Scoper implementation. So lets keep this issue open.
